src/style.css

```
@@ -1,3 +1,86 @@
+html {
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --hover-link-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  padding: 0 50px;
+  height: auto;
+  max-width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  display: block;
+  margin: 0;
+  padding: 0;
+}
+
+.img_logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  margin: 0;
+  padding: 0;
+}
+
+.nav_list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+}
+
+.link {
+  margin: 0;
+  padding: 0;
+  font-family:;
+}
+
+.nav_link {
+  display: inline-block;
+  padding: 0;
+  margin-left: 20px;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  color: #000;
+  height: 60px;
+  line-height: 60px;
+  position: relative;
+}
+
+.nav_link:hover {
+  color: var(--hover-link-color);
+}
+
+.is-active {
+  margin: 0;
+  color: var(--hover-link-color);
+}
```

```suggestion
.nav__item:first-child {
margin-left: 0;
}

.is-active {
  color: var(--hover-link-color);
}
```
better to just delete the margin-left in first li tag